applications/allocation_result_mapper.py

```diff
@@ -20,22 +20,23 @@ def to_events(self):
                 application_event_schedule = ApplicationEventSchedule.objects.get(
                     pk=allocated_event.occurrence_id
                 )
-                ApplicationEventScheduleResult.objects.update_or_create(
+
+                ApplicationEventScheduleResult.objects.create(
```

Comment:
I think changing this to create only can introduce duplicate results :thinking:  I noticed that at start of the allocation this should get deleted => Would it still be safer to stick with the update_or_create?

Reply:
Maybe it would be safer. Was thinking if we want it to fail on duplicates, because it's basically an error (not 100% sure how django handles one to one situations where more than 2 refer to it, does it actually fail on create or just pass through without error)?

Reply:
django throws an error if duplicate entry is tried to save to db, so perhaps my duplicate result worry wasn't that a big deal. we